vps/repository.go

```diff
@@ -164,8 +164,14 @@ func (r *Repository) GetUsage(vpsName string, usageTypes []UsageType, period Usa
 	for _, usageType := range usageTypes {
 		types = append(types, string(usageType))
 	}
-	requestBody := vpsUsageRequest{Types: strings.Join(types, ","), UsagePeriod: period}
```

Comment:
I think you can remove the `vpsUsageRequest` struct, as it shouldn't be used anymore